src/darwin/Framework/CHIP/CHIPOnboardingPayloadParser.h

```diff
@@ -0,0 +1,39 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class CHIPSetupPayload;
+
+typedef NS_ENUM(NSUInteger, CHIPOnboardingPayloadType) {
+    CHIPOnboardingPayloadTypeQRCode = 0,
+    CHIPOnboardingPayloadTypeManualCode,
+    CHIPOnboardingPayloadTypeNFC,
+    CHIPOnboardingPayloadTypeTomatoEnhanced,
```

Comment:
Suggestion: How about `CHIPOnboardingPayloadTypeAdmin` instead. We don't know what this is going to be called longer term but Admin seems neutral enough and not quite as arbitrary as "Tomato"

Reply:
+1 to that :)

Reply:
yep can do that